### PR TITLE
Fix leading and trailing whitespace issue

### PR DESCRIPTION
**Pull request by Edfernape**
Fixes https://trello.com/c/GwzwpWNs/141-question-category-does-not-strip-leading-and-trailing-whitespace

### CODE REVIEW

File: front-end/peer-prep/src/util/util.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (huizhuansam) [inline front-end/peer-prep/src/util/util.tsx]: Is react import required here? Not very familiar with react, can @Acerizm verify?

File: front-end/peer-prep/src/components/QuestionsPage/QuestionCreator/index.tsx [hunk 2]
```diff
@@ -164,8 +165,9 @@ const QuestionCreator = () => {
                                 helperText={duplicateCategoryErrorText}
                                 onKeyDown={(event) => {
                                     if (event.key === "Enter" && categoryBuffer !== "") {
-                                        if (!currentCategories.includes(categoryBuffer)) {
-                                            dispatch(QuestionSlice.updateCurrentCategories(categoryBuffer))
+                                        // dispatch(QuestionSlice.updateCurrentCategories(TrimWhitespace(categoryBuffer)))
+                                        if (!currentCategories.includes(TrimWhitespace(categoryBuffer))) {
```
Comment (huizhuansam) [inline front-end/peer-prep/src/components/QuestionsPage/QuestionCreator/index.tsx]: This doesn't justify creating a utility file just for a 1 liner function.

**Review by huizhuansam — CHANGES_REQUESTED** [review]

File: front-end/peer-prep/src/util/util.tsx (hunk 1 above)
Comment (Acerizm) [reply front-end/peer-prep/src/util/util.tsx]: Based on this file don't need

**Review by huizhuansam — APPROVED** [review]
LGTM

**Pull request merged**